src/main/java/racingcar/ResultView.java

```diff
@@ -0,0 +1,13 @@
+package racingcar;
+
+import java.util.ArrayList;
+
+public class ResultView {
+    public void showCurrentState(ArrayList<Car> cars) {
+        for (Car car : cars) {
+            String dashes = "-".repeat(car.getCurrentDistance());
```

Comment:
```suggestion
            String dashes = "-".repeat(car.결과? or 전진한갯수?);
```

의미를 좀 더 분명하게 하면 좋을것 같아요.
객체를 능동적으로 사용할 수 있도록 객체에게 의미를 확실하게 전달해보는건 어떨까요? 🤔 

<img width="551" alt="스크린샷 2023-04-13 오전 10 46 27" src="https://user-images.githubusercontent.com/12059592/231625145-0dda09ac-b048-483b-aee9-620993304236.png">